modules/standard/IO.chpl

```diff
@@ -4398,6 +4401,19 @@ proc channel.close() {
   if e then this._ch_ioerror(e, "in channel.close");
 }
 
+// TODO -- we should probably have separate c_ptr ddata and ref versions
```

Comment:
I can fix the comment - but it would enable better Chapel compile-time checking that the x argument has a valid type. Right now, if you pass an unspported type to x, you will fail in C compilation or have core-dumps.